RTN-008.tex

```diff
@@ -62,114 +63,113 @@
+%In \citeds{DMTN-107} the DM-SST suggests that incremental template generation in the first year of LSST operations provides the best balance of early template availability and artifact rejection.
+In this scenario new templates are regularly generated throughout the year whenever sufficient imaging becomes available in sky areas and filters without templates \citedsp{RTN-011}.
+%Having preexisting templates will allow standard alert processing in GW TOO fields, easing the operational impacts of TOO observations.
 
 Additionally, evidence of variability prior to the GW trigger provides one of the most effective ways to rule out unassociated sources in the GW localization region \citep[e.g.,][]{2019GCN.24223....1C, 2019GCN.26430....1S}.
 At LSST depths there will be many unrelated events, and little recent prior imaging of sufficient depth from other surveys that can constrain the explosion date.
-Already today at DECam followup depths the number of unrelated events is straining the ability of the worldwide community to obtain classification spectra \citep[e.g.,][]{2019ApJ...881L..16A}.
 
 In the steady state survey, LSST's own history will provide substantial filtering: simply selecting \DIAObjects created after the the trigger will exclude most unrelated contaminants.
 In early operations, however, this history will not be available, so not only distant supernovae but also variable stars and AGN will confound efforts to identify plausible GW counterparts.
 
-For this reason it will be very helpful for LSST to have reliable standard image differencing processing---and hence history---as soon as is practical, providing further impetus for incremental template building.
+For this reason it will be very helpful for Rubin to have reliable standard image differencing processing---and hence history---as soon as is practical, providing further impetus for incremental template building.
 While custom processing and association could be used to derive this history (\S \ref{sec:processing}) it would require substantial additional effort relative to LSST's standard processing.
 
 If TOO observations are expected to be conducted in a subset of LSST's filters, any observations intended to maximize template coverage could prioritize obtaining images in those bands.
 
 \subsection{Conduct bespoke image differencing on-project} \label{sec:processing}
 
-Finally, we suggest that the operations team devote specific effort to custom processing and/or analysis of TOO observations.
-This processing is most critical if the raw data cannot be made available to the community in a timely manner (\S \ref{sec:data}) and pre-existing templates are not available for the fields to be observed (\S \ref{sec:templates}).
+We suggest that the Rubin commissioning and operations teams devote specific effort to custom processing and analysis of TOO observations.
```

Comment:
See above - I don't disagree with the value of doing this but  this is an upscope with respect to our construction requirements.